packages/app-builder-lib/src/options/PlatformSpecificBuildOptions.ts

```diff
@@ -143,6 +143,11 @@ export interface ReleaseInfo {
    * The release date.
    */
   releaseDate?: string
+
+  /**
+   * Vendor specific information.
+   */
+  vendor?: Record<string, unknown>
```

Comment:
I think for this it needs to be: `Record<string, unknown> | null`?

(Honestly, not sure why `| null` exists in so many places when the optional `?` is already leveraged, but I'm assuming it's for doc generation?)

Reply:
Sure thing!
```suggestion
  vendor?: Record<string, unknown> | null
```